pure_pursuit_wrapper/src/pure_pursuit_wrapper_node.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018-2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <ros/ros.h>
+#include "pure_pursuit_wrapper/pure_pursuit_wrapper.hpp"
+#include <ros/console.h>
+
+#include <message_filters/subscriber.h>
+#include <message_filters/time_synchronizer.h>
+#include <message_filters/sync_policies/approximate_time.h>
+
+int main(int argc, char** argv) {
+  // if (ros::console::set_logger_level(ROSCONSOLE_DEFAULT_NAME, ros::console::levels::Info)) { 
+  //   ros::console::notifyLoggerLevelsChanged();
+  // }
+
+  ros::init(argc, argv, "pure_pursuit_wrapper_node");
+  ros::NodeHandle nh("~");
```

Comment:
You should only use private node handles ("~") for parameters. Topics should use base names ""

Reply:
fixed